tests/test_energy.py

```diff
@@ -0,0 +1,44 @@
+"""
+this is to test the eneryg of H2O with sto-3g basis
+
+"""
+
+from qm7.scf import SCF
+import psi4
+import argparse
+
+def test_energy():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('--molecule', '-m', type=str, default='tests/water', metavar='',
+                    help='Molecule file name (default: water)')
+    parser.add_argument('--basis', '-b', type=str, default='sto-3g', metavar='',
+                    help='Basis set selection (defult: sto-3g)')
+    parser.add_argument('--nocc', '-n', type=int, default=5, metavar='',
+                    help='Number of occupied orbitals (defult: 5)')
+    parser.add_argument('--max_iter', '-i', type=int, default=50, metavar='',
+                    help='Max number of iterations (defult: 50)')
+    parser.add_argument('--econv', type=float, default=1.e-6, metavar='',
+                    help='Energy convergence value (default: 1.e-6)')
+    parser.add_argument('--dconv', type=float, default=1.e-6, metavar='',
+                    help='Density convergence value (default: 1.e-6)')
+    parser.add_argument('--damping', action='store_true', default=False,
+                    help='Use damping (default: False)')
+    parser.add_argument('--dampvalue', '-dv', type=float, default=0.2, metavar='',
+                    help='Damping value (default: 0.2)')
+    parser.add_argument('--dampstart', '-ds', type=int, default=5, metavar='',
+                    help='Iteration to start damping (default: 5)')
+    parser.add_argument('--DIIS', '-D', action='store_true', default=False,
+                    help='Iteration to start damping (default: 5)')
+    
+    parser.add_argument('--verbose', '-v', type=str, action=None, default="false")  # For tests
```

Comment:
We don't need any of the parser code here. You can just give the arguments you want and run the scf as:
```python
scf = SCF(args)
scf.initialize()
scf.solve()
```

Take a look at here: [How to write tests for argparse](https://stackoverflow.com/questions/18160078/how-do-you-write-tests-for-the-argparse-portion-of-a-python-module)